spec-tests/src/main/java/org/candlepin/spec/bootstrap/client/ApiClients.java

```diff
@@ -129,6 +129,43 @@ public static ApiClient bearerToken(String token) {
         return new ApiClient(CLIENT_FACTORY.createBearerTokenAuthClient(token));
     }
 
+    /**
+     * Returns a client to make API calls with, that uses an OAuth authentication.
+     *
+     * @param oauthConsumer an OAuth consumer
+     * @param oauthSecret an OAuth consumer secret
+     * @return a client to make API calls with, that uses an OAuth authentication
+     */
+    public static ApiClient oauth(String oauthConsumer, String oauthSecret) {
+        return new ApiClient(CLIENT_FACTORY.createOauthClient(oauthConsumer, oauthSecret));
+    }
+
+    /**
+     * Returns a client to make API calls with, that uses an OAuth user authentication.
+     *
+     * @param oauthConsumer an OAuth consumer
+     * @param oauthSecret an OAuth consumer secret
+     * @param username a user username
+     * @return a client to make API calls with, that uses an OAuth user authentication
+     */
+    public static ApiClient oauthUser(String oauthConsumer, String oauthSecret, String username) {
+        return new ApiClient(CLIENT_FACTORY.createOauthClient(oauthConsumer, oauthSecret)
+            .addDefaultHeader("cp-user", username));
+    }
+
+    /**
+     * Returns a client to make API calls with, that uses an OAuth consumer authentication.
+     *
+     * @param oauthConsumer an OAuth consumer
```

Comment:
The term "consumer" is overloaded in this context (OAuth and Candlepin both have their own definitions of a consumer), so clarification is needed to differentiate them.

Reply:
is that enough, or should I add something else I'm not sure about the acting like sentence

Reply:
The "acting like" sentences are extraneous and somewhat confusing, but the addition of "Candlepin" on the param is sufficient here.